sdk/src/main/java/com/mapbox/maps/ViewAnnotationManagerImpl.kt

```diff
@@ -143,6 +146,69 @@ internal class ViewAnnotationManagerImpl(
     return renderThread.viewAnnotationMode
   }
 
+  override fun cameraForAnnotations(
```

Comment:
It makes sense to have it in mapboxMap, but i think mapboxMap class will be crowded with all the calculations that we do to calculate bounds and edgeInsets. although, we can use extension functions for that. 

cc @maios how does this sound from ios side?

Reply:
imo it is better to house this method in `ViewAnnotationManager`, I think `ViewAnnotationManager`is created to have a separated set of APIs relating to view annotations, and to offload some interfaces from MapboxMap so that it does not become too crowded, what do you think @mapbox/maps-ios ?